packages/combobox/src/Menu.styles.ts

```diff
@@ -0,0 +1,80 @@
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { fontFamilies, typeScales, Mode } from '@leafygreen-ui/tokens';
+import { ComboboxSize } from './Combobox.types';
+import { menuItemHeight, menuItemPadding } from './constants';
+
+/**
+ * Menu styles
+ */
+
+export const popoverStyle = (width = 384) => css`
+  border-radius: 4px;
+  width: ${width}px;
+
+  & > * {
+    border-radius: inherit;
+  }
+`;
+
+export const menuBaseStyle = css`
+  position: relative;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  font-family: ${fontFamilies.default};
+  border-radius: inherit;
+  overflow: auto;
+  scroll-behavior: smooth;
+  min-height: ${menuItemHeight}px;
+`;
+
+export const menuModeStyle: Record<Mode, string> = {
+  [Mode.Light]: css`
+    color: ${uiColors.gray.dark3};
+    background-color: ${uiColors.white};
+    box-shadow: 0px 3px 7px rgba(0, 0, 0, 0.25);
+  `,
+  [Mode.Dark]: css``, // TODO: DarkMode
+};
+
+export const menuList = css`
+  position: relative;
+  margin: 0;
+  padding: 0;
+`;
+
+export const menuMessageBaseStyle = css`
+  font-family: inherit;
+  display: inline-flex;
+  align-items: center;
+  gap: 8px;
+  & > svg {
+    width: 1em;
```

Comment:
`font-size` for this element is being defined in `menuMessageSizeStyle`. We want the icon to be either `13px` or `16px` depending on the Size variant

Reply:
Seems a bit brittle to use em units this way when we don't control the context outside of the component. Though it's a bit more work, I feel we should probably be applying the size CSS specifically based on the variant, and probably also applying it through a className rather than a child selector.